### PR TITLE
quickstart notebooks folder base

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

File: quickstart/llms_get_started.ipynb [hunk 1]
```diff
@@ -0,0 +1,1061 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# model instantiation \n",
+    "llm = SambaNovaCloud(\n",
+    "    model='llama3-405b',\n",
```
Comment (snova-petrojm) [inline quickstart/llms_get_started.ipynb:60]: I suggest using Meta-Llama-3.1-405B-Instruct instead

File: quickstart/llms_get_started.ipynb [hunk 2]
```diff
@@ -0,0 +1,1061 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# model instantiation \n",
+    "llm = ChatSambaNovaCloud(\n",
+    "    model= \"llama3-405b\",\n",
```
Comment (snova-petrojm) [inline quickstart/llms_get_started.ipynb:417]: Same comment here as above

**Review by snova-petrojm — COMMENTED** [review]
Can you add a cell at the beginning and mention what are the required packages to be installed? I think the following are needed 
!pip install python-dotenv==1.0.0
!pip install requests
!pip install langchain==0.2.16
!pip install langchain-core==0.2.38
!pip install langchain-community==0.2.16
!pip install sseclient-py==1.8.0

**Comment by snova-petrojm** [comment]
Added my comments above for llms_get_started.ipynb. Will review the other notebook tomorrow. Thanks

File: quickstart/llms_get_started.ipynb (hunk 1 above)
Comment (snova-jorgep) [reply quickstart/llms_get_started.ipynb:60]: done

File: quickstart/llms_get_started.ipynb (hunk 2 above)
Comment (snova-jorgep) [reply quickstart/llms_get_started.ipynb:417]: done

**Review by snova-petrojm — APPROVED** [review]
Getting "Server is overloaded" when running the last cell of  multimodal_get_started.ipynb with 2 images. 

Besides this, LGTM, thanks, Jorge.

**Comment by snova-jorgep** [comment]
removed cell with multiple images prompt for now

**Pull request merged**